src/app/pages/about/about.component.html

```diff
@@ -1,3 +1,238 @@
-<div class="container">
-  <h1 align=center>About works!</h1>
+<div id="about-us" class="jumbotron">
+  <h1 id="title">About Us</h1>
+  <div class="row justify-content-center">
+    <div class="col-md-4 vertical-center">
+      <img class="img-fluid" alt="pink cloud graphic" src="../../../assets/images/logo_reg_transparent.png"/>
+    </div>
+    <div class="col-md-4 mt-3 mt-md-0 vertical-center">
+      <p class="text-justify">
+        We’re a group of undergraduate university students who remember exactly what it was like back in high school
+        (The Woodlands Secondary School, to be exact, woot woot),
+        and we especially remember what those last few months of senior year meant to many of us. With the COVID-19
+        pandemic leading to lockdowns and social distancing, we wanted to give a way for graduating students (and
+        others!) to be able to send each other messages in a medium that they can cherish with them as they continue on
+        in their journeys.
+      </p>
+    </div>
+  </div>
 </div>
+
+<div class="container">
+  <div class="mb-3" id="meet-the-team">
+    <h1 class="mb-4" id="meet-the-team-title">Meet the Team!</h1>
+
+    <!-- Arvind's Row-->
+    <div class="row d-flex">
+
+      <!-- Arvind's Image -->
+      <div class="col-md-2 order-md-2">
+        <img src="{{Arvind.regularLink}}" class="img-fluid" (mouseenter)="Arvind.showSilly = !Arvind.showSilly"
+          *ngIf='Arvind.showSilly else ArvindSilly'>
+        <ng-template #ArvindSilly>
+          <img src="{{Arvind.sillyLink}}" class="img-fluid" (mouseleave)="Arvind.showSilly = !Arvind.showSilly">
+        </ng-template>
+      </div>
+
+      <!-- Arvind's Blurb -->
+      <div class="col-md-7 order-md-3 mt-3 mt-md-0">
+        <p class="text-justify">Arvind is Virtrolio's ideator. He is well known for his compassion and spectacular,
+          radiant personality that always lifts the spirits of those around him. He is known to have an undying love for
+          musicals and mathematics. Taking that love of Mathematics to the extreme, he is currently studying Mathematics
+          at the University of Waterloo. In his spare time, he loves to practice his nunchucks and his tiechwaondoe
+          (especially <s>on</s> with his beloved friends)!</p>
+      </div>
+
+      <!-- Arvind's Contact Info. -->
+      <div class="col-md-3 order-md-1">
+        <p><strong>Contact Arvind:</strong></p>
+        <ul>
+          <li>Instagram: @_airwind</li>
```

Comment:
Arvind wants his LinkedIn added:
```suggestion
          <li>Instagram: @_airwind</li>
          <li>LinkedIn: <a href="https://www.linkedin.com/in/arvindn01/">Arvind Nagabhirava</a></li>
```